packages/libs/react-components/src/components/Notification/styles.ts

```diff
@@ -0,0 +1,117 @@
+/* eslint @kadena-dev/typedef-var: 0 */
+
+import { IconButton } from '../..';
+import { styled } from '../../styles';
+import { Heading, Text } from '../Typography';
+
+export const colorVariant = {
+  default: {
+    backgroundColor: `$neutral2`,
+    borderColor: '$neutral6',
+    color: '$neutral6',
+  },
+  primary: {
+    backgroundColor: `$primarySurface`,
+    borderColor: '$primaryAccent',
+    color: '$primaryAccent',
+  },
+  secondary: {
+    backgroundColor: `$secondarySurface`,
+    borderColor: '$secondaryAccent',
+    color: '$secondaryAccent',
+  },
+  positive: {
+    backgroundColor: `$positiveSurface`,
+    borderColor: '$positiveAccent',
+    color: '$positiveAccent',
+  },
+  warning: {
+    backgroundColor: `$warningSurface`,
+    borderColor: '$warningAccent',
+    color: '$warningAccent',
+  },
+  negative: {
+    backgroundColor: `$negativeSurface`,
+    borderColor: '$negativeAccent',
+    color: '$negativeAccent',
+  },
+};
+
+export const expandVariant = {
+  true: { width: '100%', maxWidth: '100%' },
+  false: {},
+};
+
+export const simpleVariant = {
+  true: {
+    borderRadius: '$md',
+    borderLeftWidth: '1px',
+  },
+  false: {},
+};
+
+export const StyledNotification = styled('div', {
+  margin: '0px $sm',
```

Comment:
I dont know if we want a margin in here.
I think the positioning of a libary component should be set by the parent component in the application.

If we do want margins here: then the margins do not work well at the moment. in storybook i have a larger margin on the left than on the right